style.css

```diff
@@ -1,531 +1,643 @@
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
-
-@media only screen and (max-width: 600px) {
-  header img {
-    display: none;
-  }
-}
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;500;800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@600&display=swap');
 
 * {
-  font-family: 'Inter', sans-serif;
-  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
   margin: 0;
   padding: 0;
-  text-decoration: none;
-  box-sizing: border-box;
-  overflow-x: hidden;
 }
 
 body {
-  font-family: 'Poppins', sans-serif;
+  position: relative;
   font-family: 'Inter', sans-serif;
-  max-width: 375px;
 }
 
-header {
-  color: #fff;
-  width: 100%;
+.backgrounds {
   position: relative;
-  display: flex;
-  justify-content: end;
 }
 
-.menu {
-  padding: 1px 35px;
-  margin-top: 15px;
-  width: auto;
-}
+@media (min-width: 375px) {
+  .desktop-menu {
+    display: none;
+  }
 
-.background_image {
-  background-image: url('images/header-illsutration-mobile.png');
-  background-repeat: no-repeat;
-  background-size: 100%;
-  width: 100%;
-  height: 653px;
-}
+  .mobile-menu-icon {
+    display: flex;
+    justify-content: end;
+    padding: 20px 35px;
+  }
 
-.info {
-  font-size: 40px;
-  display: block;
-  align-items: center;
-  text-align: center;
-  color: #172b4d;
-  padding-top: 44px;
-  margin-top: 5px;
-}
+  .home {
+    background: url(./images/header-illsutration-mobile.svg) no-repeat;
+    background-position: 100% 0;
+    height: 100vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-size: 100%;
+  }
```

Comment:
- [ ] Nice work so far 👍🏿 on implementing the required changes as suggested by the previous reviewer but on inspection, I notice the responsive issues have not been resolved just yet. I kindly suggest you wrap the header section in a `<div>` tag and adjust it using margin and padding, it will solve the issues 😉